docs/content/python/getting-started/quickstart/index.md

```diff
@@ -0,0 +1,147 @@
+- What are they trying to do? (the "action")
+- What are they doing it to? (the "resource")
+
+In Oso, these pieces of information are passed to the `{{% exampleGet "isAllowed" %}}` call: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
+`{{% exampleGet "isAllowed" %}}` will return `True` or `False`, and your application can choose how to enforce that decision.
+
+That enforcement can happen in the request handler, at the database layer, or in middleware — here, we've chosen to enforce in the request handler.
+Here's {{% exampleGet "example_app" %}} route that displays a page if this user is allowed to read the associated page.
+
+{{% exampleGet "app_code" %}}
+
+
+Oso denies requests unless you explicitly tell it to accept that sort of request.
+You can tell Oso what requests to accept by providing it with a file full of rules, which we call a policy.
+
+## Write an authorization policy
+Here is a typical policy, written in our declarative language, **Polar**.
+It lets any actor with the role `"user"` read a page, but only actors with the role `"admin"` can write to a page.
+
+We can load our example policy from a file with the extension `.polar`.
+
+{{% exampleGet "load_policy" %}}
+
+Here's the authorization.polar file:
+
+```polar
+allow(actor, action, resource) if
+    role_allow(actor, action, resource);
+
+actor_has_role_for_resource(actor, role_name, resource) if
+    role in actor.{{% exampleGet "getroles" %}}() and
+    role_name = role.name and
+    resource = role.resource;
+
+resource(_type: Page, "page", actions, roles) if
+    actions = ["read", "write"] and
+    roles = {
+        user: {
+            permissions: ["read"]
+        },
+        admin: {
+            permissions: ["write"],
+            implies: ["user"]
+        }
+    };
+ ```
+
+The `allow` rule is the top-level rule that we use to say who can do what in our application.
+In this case, we are delegating to Oso's builtin `role_allow` rule which implements all the
+authorization logic for role-based access control based on the data we provide in `actor_has_role_for_resource`
+and `resource`.
+
+An `actor_has_role_for_resource` rule looks up the roles assigned
+to the actor. Each role is parameterized by the `role_name` and the
+`resource` the role is associated with.
+The Oso builtin policy depends on this rule, so this is required.
+
+A `resource` rule governs access to a specific resource in your app — for instance, a page, an object, a route, or a database entry.
+Here, there are two possible actions that can be done to a `Page`: `"read"` and `"write"`.
+Users can read, but not write, to a `Page`.
+Admins are allowed to write to a `Page`.
+
+The line `implies: ["user"]` says that admins can also do anything users are allowed to do; that is, read a `Page`.
+
+There's much more you can do with Polar, including defining parent-child relationships — we're just scratching the surface here.
+
+## Calling back into your {{% lang %}} code
```

Comment:
This content in this section feels way more about how to write an `actor_has_role_for_resource` predicate than about "Calling back into your Python code"

Some suggestions:

- Passing application roles into Oso
- Passing application-managed roles into Oso
- Fetching user roles via `actor_has_role_for_resource`

Reply:
The reason this is a section header is we wanted to emphasize that objects in Polar have the same attributes and methods available as the types passed in from Python. This is a big part of the old quickstart, and a key component of the product. 

However I agree that the content of the section is more about implementing that predicate.